Google.PowerShell.IntegrationTests/Compute/Compute.GceInstance.Tests.ps1

```diff
@@ -198,6 +213,22 @@ Describe "Add-GceInstance" {
         }
     }
 
+    It "should work with -CustomCpu and -CustomMemory" {
+        $r = Get-Random
+        $instance = "gcps-instance-create-$r"
+        $instanceConfig = New-GceInstanceConfig -Name $instance -DiskImage $image -CustomCpu 2 -CustomMemory 2048
+
+        try {
+            Add-GceInstance -Project $project -Zone $zone -Instance $instanceConfig
```

Comment:
I feel like it would be better to test the "ByValues" version of this, as we already have a test for `New-GceInstanceConfig` with custom values.